mate_menu/easybuttons.py

```diff
@@ -408,7 +408,7 @@ def filterText( self, text ):
         return True
 
     def strip_accents(self, value):
-        return unidecode.unidecode(value)
+        return value
```

Comment:
Now the strip_accents method is unneeded and you can remove it and all its calls in this file.

Reply:
And you can also remove unidecode dependency as it is no longer used.